### PR TITLE
jasper: add 2.0.26 + provide official CMake variables in cmake_find_package generator

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
see https://cmake.org/cmake/help/latest/module/FindJasper.html for official find module file.

File: recipes/jasper/all/conanfile.py [hunk 1]
```diff
@@ -92,11 +95,43 @@ def package(self):
...
                 if os.path.basename(dll_file).startswith(("concrt", "msvcp", "vcruntime")):
                     os.unlink(dll_file)
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_file_rel_path)
+        )
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file):
+        content = textwrap.dedent("""\
+            if(DEFINED Jasper_FOUND)
```
Comment (SSE4) [inline recipes/jasper/all/conanfile.py:105]: seems to be one more repeating pattern - mapping CMake variables, besides one for creating alias targets

**Review by SSE4 — APPROVED** [review]

File: recipes/jasper/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/jasper/all/conanfile.py:105]: In the conan issues for this, it was not desired to add this ... perhaps proposing a new tool would be possible?

**Review by prince-chrismc — APPROVED** [review]

File: recipes/jasper/all/conanfile.py (hunk 1 above)
Comment (SSE4) [reply recipes/jasper/all/conanfile.py:105]: yes, new tool using build modules

**Review by uilianries — APPROVED** [review]

**Pull request merged**